### PR TITLE
Update Rust quickstart

### PR DESCRIPTION
**Pull request by xinnks**

### CODE REVIEW

File: sdk/rust/quickstart.mdx [hunk 1]
```diff
@@ -11,41 +11,49 @@ In this Rust quickstart we will learn how to:
...
 ```bash
-cargo add libsql-client
+cargo add libsql
 ```
 
   </Step>
 
   <Step title="Set environment variables">
-    Next export your `LIBSQL_CLIENT_URL` and `LIBSQL_CLIENT_TOKEN`:
+    Next export your `TURSO_DATABASE_URL` and `TURSO_AUTH_TOKEN`:
```
Comment (notrab) [inline sdk/rust/quickstart.mdx:25]: I like that we encourage this. Perhaps an improvement we can make with other quickstarts.

File: sdk/rust/quickstart.mdx [hunk 2]
```diff
@@ -11,41 +11,49 @@ In this Rust quickstart we will learn how to:
 - Execute a query using SQL
 
 <Steps>
-  <Step title="Install libsql-client">
+  <Step title="Install libsql crate">
```
Comment (notrab) [inline sdk/rust/quickstart.mdx:14]: @xinnks let's make sure we include the new Snippet for obtaining database credentials. You might need to rebase with `main`.

File: sdk/rust/quickstart.mdx (hunk 1 above)
Comment (xinnks) [reply sdk/rust/quickstart.mdx:25]: Consistency in the docs and references is good.

**Review by notrab — APPROVED** [review]
Let's get this merged if @LucioFranco is good with it. Perhaps we could enhance it further with the embedded replica story? If you see the Go example you'll see the different paths for that one.

We need to later update the repo README (which I'm already working on in another branch) so it follows our new README style linking to this doc.

**Pull request merged**